### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by bengerman13**
Using python 3.4 was painful, so I updated the container to Bionic -- now we get python 3.6.

This also means that we can add `cf-uaac` back in, since the ruby bug that was preventing its install doesn't seem to affect Bionic.

### CODE REVIEW

**Comment by bengerman13** [comment]
(cc: @mxplusb )

File: task/scripts/build.sh [hunk 1]
```diff
@@ -8,9 +8,9 @@ set -e
 source ./config.sh
 
 # Install current postgres
-apt-get update && apt-get -y -q install wget
+apt-get update && apt-get -y -q install wget lsb-release gnupg2 tzdata
```
Comment (mxplusb) [inline task/scripts/build.sh]: why are you removing `gnupg2`?

Comment (bengerman13) [reply task/scripts/build.sh]: I'm adding it, not removing it

Comment (mxplusb) [reply task/scripts/build.sh]: Er, right, lol sorry. All good then.

**Pull request merged**